man/oomd.conf.xml

```diff
@@ -48,28 +48,30 @@
 
     <variablelist class='config-directives'>
       <varlistentry>
-        <term><varname>SwapUsedLimitPercent=</varname></term>
-
-        <listitem><para>Sets the limit for swap usage on the system before <command>systemd-oomd</command> will
-        take action. If the percentage of swap used on the system is more than what is defined here,
-        <command>systemd-oomd</command> will act on eligible descendant cgroups, starting from the ones with the
-        highest swap usage to the lowest swap usage. Which cgroups are monitored and what
-        action gets taken depends on what the unit has configured for <varname>ManagedOOMSwap=</varname>.
-        Takes a percentage value between 0% and 100%, inclusive. Defaults to 90%.</para></listitem>
+        <term><varname>SwapUsedLimit=</varname></term>
+
+        <listitem><para>Sets the limit for swap usage on the system before <command>systemd-oomd</command>
+        will take action. If the fraction of swap used on the system is more than what is defined here,
+        <command>systemd-oomd</command> will act on eligible descendant cgroups, starting from the ones with
+        the highest swap usage to the lowest swap usage. Which cgroups are monitored and what action gets
+        taken depends on what the unit has configured for <varname>ManagedOOMSwap=</varname>.  Takes a value
+        specified in percent (when suffixed with "%"), permille ("‰") or permyriad ("‱"), between 0% and
+        100%, inclusive. Defaults to 90%.</para></listitem>
```

Comment:
Maybe underline that a plain fraction is also OK: "Takes a fraction (when specified without a suffix), or a value in percent (when suffixed with "%"), permille ("‰") or permyriad ("‱"), between 0% and 100%, inclusive. Defaults to 90%."

Reply:
hmm, wa? that's a misunderstanding, plain fractions are currently not OK. (I mean, we can add them, but right now there's no code for that...)

Reply:
(i wonder if we add them, if people might make mistakes like assume that omitting the `%` suffix still means percent)

Reply:
Oh, bummer. Let's leave it as it then.